lib/compat/wordpress-6.2/html-api/class-wp-html-tag-processor.php

```diff
@@ -61,9 +61,9 @@
  * | Goal                                                      | Query                                                                      |
  * |-----------------------------------------------------------|----------------------------------------------------------------------------|
  * | Find any tag.                                             | `$tags->next_tag();`                                                       |
- * | Find next image tag.                                      | `$tags->next_tag( [ 'tag_name' => 'img' ] );`                              |
- * | Find next tag containing the `fullwidth` CSS class.       | `$tags->next_tag( [ 'class_name' => 'fullwidth' ] );`                      |
- * | Find next image tag containing the `fullwidth` CSS class. | `$tags->next_tag( [ 'tag_name' => 'img', 'class_name' => 'fullwidth' ] );` |
+ * | Find next image tag.                                      | `$tags->next_tag( array( 'tag_name' => 'img' ) );`                              |
+ * | Find next tag containing the `fullwidth` CSS class.       | `$tags->next_tag( array( 'class_name' => 'fullwidth' ) );`                      |
+ * | Find next image tag containing the `fullwidth` CSS class. | `$tags->next_tag( array( 'tag_name' => 'img', 'class_name' => 'fullwidth' ) );` |
```

Comment:
@aristath looks like the formatting of the table was messed up here during this change. would you mind updating to fix it?

as a side-note, I know we had held a discussion about the use of the shorthand syntax in comments vs. in code. I guess there are still some differences in opinion around about that. would have been nice to be at least pinged in this patch so we could have caught wind of the change.

note that there are a couple existing uses of array shorthand syntax in comments in rest-api.php, as well as prevalent use in wp-includes/Requests. I suspect you will want to remove those too.

Reply:
I don't think documentation should include code examples that cannot be used. Wether it is in a comment or not.

Reply:
@carolinan my comment is not arguing whether this was a positive or negative change, but pointing out a process note:
 - it would be courteous and cooperative to include people who have recently been doing a lot of work within a file to poke them in a PR before rapidly proposing and merging changes.
 - the style changes in this PR are not comprehensive, leaving in place existing examples of the style being changed, which raises the question "how do we decide if we change the style of existing accepted code?"

Reply:
Yes you are right.

Reply:
> looks like the formatting of the table was messed up here during this change. would you mind updating to fix it?

Thank you for catching that! I submitted a followup on https://github.com/WordPress/gutenberg/pull/48816 👍 

> it would be courteous and cooperative to include people who have recently been doing a lot of work within a file to poke them in a PR before rapidly proposing and merging changes.

You are 100% correct. Please accept my apologies, it didn't cross my mind.

> I know we had held a discussion about the use of the shorthand syntax in comments vs. in code. I guess there are still some differences in opinion around about that.

Yeah... Personally, I prefer the short syntax, and I wish we could use it everywhere. That's what I always use when writing code. However, the official WP Coding Standards recommend using the long-array syntax. As a result, when someone copy-pastes code from these examples/docs and tries to use them in a projects that uses the WordPress Coding Standards (like for example Gutenberg), the IDE throws CS notices/errors about the use of the short array syntax, and automated tests won't pass. That's why this change was made...

Reply:
thanks all